src/node.cpp

```diff
@@ -318,6 +330,24 @@ int main(int argc, char * argv[]) {
         end_scan_time = ros::Time::now();
         scan_duration = (end_scan_time - start_scan_time).toSec();
 
+        // Throttle Set async
+        peanut_common::HardwareStatus msg;
+        if (!checkRPLIDARHealth(drv)) {
+            msg.status = peanut_common::HardwareStatus::STATUS_DISCONNECTED;
+            msg.status_msg = "Error Undefined";
+        }
+        else{
+            msg.status = peanut_common::HardwareStatus::STATUS_READY;
+            msg.status_msg = "Status OK";
+        }
+
+        ros::Time now = ros::Time::now();
```

Comment:
Messages on topics occasionally get lost, I think you'd want to constantly publish the status at some low frequency ~1 Hz